manifest/schema2/manifest.go

```diff
@@ -132,3 +132,16 @@ func (m *DeserializedManifest) MarshalJSON() ([]byte, error) {
 func (m DeserializedManifest) Payload() (string, []byte, error) {
 	return m.MediaType, m.canonical, nil
 }
+
+// ConfigType represents the type of configuration inside the manifest based on
+// on the media type of the config. Return values are "image" for the default
```

Comment:
The only reason I added it here is because that is where the configurations are defined. I wasn't thinking of the classification of the config types as policies, only their enforcement. The set of classes should be constrained and updated along with the manifest package.